src/resources/projects/website/listing/item-default.ejs.md

```diff
@@ -31,44 +31,64 @@ return value;
 
 let value = readField(item, field);
 if (value !== undefined) {
-print(`<div class="metadata-value listing-${field}">${listing.utilities.outputLink(item, field, value)}</div>`);  
+print(`<div class="metadata-value listing-${field}">${listing.utilities.outputLink(item, field, value)}</div>`);
  }
 }
 %>
 
-<div class="quarto-post image-<%= imageAlign %>" <%= listing.utilities.metadataAttrs(item) %>>
+::: {.quarto-post .image-<%= imageAlign %> <%= listing.utilities.metadataAttrs(item) %>}
```

Comment:
Switching to div as raw inline/block content leads to parsing issues.

Reply:
The line is not taken as a line and is broken when using raw inline or block.
It mostly breaks the filter/sort feature which is the most obvious issue.
I'm am not yet sure to what's causing this.